src/config.py

```diff
@@ -21,6 +21,18 @@ class Settings(BaseSettings):
     db_host: str
     db_scheme: str = "postgresql+asyncpg"
     conn: PostgresDsn | None = None
+    """
+    upload_fs_protocol: to be used with fsspec, and s3fs
+    `file` is for local file system
+    `s3` is for AWS S3
+    """
+    upload_fs_protocol: str
```

Comment:
Should we make this an enum and default to file?

Reply:
yes we should, I'll get on that